attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>This text is similar to the initial introduction I recommend merging the two in the beginning and just refer back to it in this section.</p><p> </p><p> </p><p><strong>Remove repetition</strong></p><p> </p><p>&gt;In this section, we'll implement a Transformer based architecture in Haiku for a simpler task than translation: inverting a sentence. Each block will be built up from scratch and will be able to do more complex tasks.  </p><p>In this part of the tutorial we'll implement a Transformer based architecture in Haiku for a simpler task then translation: inverting a sentence. </p><p> </p><p>&gt;In this section, we'll implement a Transformer based architecture in Haiku for a simpler task than translation: inverting a sentence. Each block will be built up from scratch and will be able to do more complex tasks.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>